### ISSUE TITLE
Enhance binding for actions

### ISSUE THREAD

**Issue by egoist** [egoist/revue#4]
I think Revue should wrap all actions in a thunk when binding to `Vue.prototype`, thus, we can do things like:

```js
this.$actions.addTodo(todo);
// instead of
this.$store.dispatch(this.$actions.addTodo(todo))
```

**Comment by egoist** [comment]
or provide a helper:

```js
import {wrap} from 'revue'

import todos from './actions/todo'
const todoActions = wrap(todos)

this.$store.dispatch(todo.addTodo({}))
// is equal to
todoActions.addTodo()
```

**Comment by egoist** [comment]
The helpers seems like a better choice, I'll implement this first.

**Comment by egoist** [comment]
closed since 2.0-alpha was released. 💥
